FileStorage/FileStorage/Program.cs

```
@@ -1,12 +1,62 @@
 ﻿using System;
+using FileStorage.Enums;
+using FileStorage.Models;
 
 namespace FileStorage
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            try
+            {
+                var flags = ConsoleFlagParser.Parse(args);
+                if (flags.ContainsKey(StorageFlags.Login) && flags.ContainsKey(StorageFlags.Password))
```

could you throw an exception if there is no required flag in parameters - instead of wrapping it into `if`